rest-api-spec/src/main/resources/rest-api-spec/api/fleet.global_checkpoints.json

```diff
@@ -32,6 +32,11 @@
         "description":"Whether to wait for the global checkpoint to advance past the specified current checkpoints",
         "default":"false"
       },
+      "wait_for_index":{
```

Comment:
Can we add a single yml test exercising this parameter to ensure it works all the way through?